test/unit/url_view.cpp

```diff
@@ -91,6 +91,12 @@ class url_view_test
                 std::exception);
         }
 
+        // url_view(string_view) no ambiguous
+        {
+            string_view s = "";
+            BOOST_TEST_NO_THROW(url_view( s ));
```

Comment:
Because the template overload has a lower priority than the string_view overload?

Reply:
Yes :)